tests/tests/swfs/avm2/xml_set_name/test.toml

```diff
@@ -0,0 +1 @@
+num_frames = 1
```

Comment:
A small nit, but I think according to the README.md in tests folder, the use of `num_frames` was deprecated and `num_ticks` is meant to be instead?